### ISSUE TITLE
Feature Request: Separable Notebooks / Partial Sync

### ISSUE THREAD

**Issue by orangelynx** [laurent22/joplin#1717]
I have been looking for a OneNote / Everlong alternative a long time with non-wysiwyg style editor / markdown support and sync support. Jopling checked most of my boxed, but there is one significant feature missing in my opinion. In OneNote, you can have separate Notebooks, for example one for private stuff (blatant example: personal medical records, e.g. "Things I need to ask my doctor the next time i see him") and another for work. Naturally, I wouldn't want the former to be synced and stored on my employer's PC, while I NEED the latter on my employers PC to work. So I want to be able to switch between both on my private device and have only the work one synced to the work pc, and perhaps the private one to my mobile phone. This is afaik currently possible with the way Joplin stores data.
Speaking of which: It would be better practice to store the user content not in AppData (and equivalent) but in the users home or documents folder. I know that Microsoft somehwhat propagates a different route these days, but having the content in the Documents folder makes it easy to access (non-hidden folder), backup and transfer.

TLDR:
- Master Notebooks (separate files) with independent sync settings would help to keep private information off company owned devices.
- Content Files should not be hidden away from the user in AppData (and such) because it makes backups more difficult and content less portable.

**Comment by subi54** [comment]
As a workaround, you can use the Portable app for private stuff and the normal one for your syncing needs.
